components/brave_wallet_ui/common/async/hardware.ts

```diff
@@ -63,3 +83,19 @@ export async function signMessageWithHardwareKeyring (apiProxy: WalletApiProxy,
   }
   return { success: false, error: getLocale('braveWalletUnknownKeyringError') }
 }
+
+export function parseLedgerDeviceError (e: any): HardwareWalletErrorType {
+  if (e instanceof LedgerTransportStatusError && (e as any).statusCode === LedgerStatusCodes.CONDITIONS_OF_USE_NOT_SATISFIED) {
+    return 'transactionRejected'
```

Comment:
Can we use `HardwareWalletErrorType.transactionsRejected` here and for the other returns?

Reply:
Will keep this for a later PR. 👍